cmd/kops/util/validate_usage.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"k8s.io/kops/pkg/featureflag"
+	"k8s.io/kops/upup/pkg/fi"
+)
+
+const (
```

Comment:
These constants are intended to be used in every util.ValidateUsage invocation. I didn't want to use loose strings for every method call. I am using those constants to populate unsupportedCommandsForVSphere just because I have them.

I not sure how I will be able to achieve that (using constants for method invocation), with the scheme that you explained above.